### PR TITLE
Add aod-writer options resdir and maxfilesize as command line options

### PR DESCRIPTION
**Pull request by pbuehler**

### CODE REVIEW

**Comment by pbuehler** [comment]
Hi @chiarazampolli, @jgrosseo,
Jan could you please approve this one?

File: Framework/Core/src/WorkflowCustomizationHelpers.cxx [hunk 1]
```diff
@@ -56,7 +56,9 @@ std::vector<ConfigParamSpec> WorkflowCustomizationHelpers::requiredWorkflowOptio
 
            // options for AOD writer
            {"aod-writer-json", VariantType::String, "", {"Name of the json configuration file"}},
+           {"aod-writer-resdir", VariantType::String, "", {"Name of the output directory"}},
            {"aod-writer-resfile", VariantType::String, "", {"Default name of the output file"}},
+           {"aod-writer-maxfilesize", VariantType::Float, 0.0f, {"Maximum size of an output file in megabytes"}},
```
Comment (chiarazampolli) [inline Framework/Core/src/WorkflowCustomizationHelpers.cxx:61]: Do you really want this as a float?

**Comment by chiarazampolli** [comment]
Ciao @pbuehler , 
Thanks! I have a minor comment. In general, though, I can approve, but my approval won't be enough to merge.
Cheers,
Chiara

File: Framework/Core/src/WorkflowCustomizationHelpers.cxx (hunk 1 above)
Comment (pbuehler) [reply Framework/Core/src/WorkflowCustomizationHelpers.cxx:61]: For testing it was good to be able to set small values (some kilobytes). That is primarilly why it is a float.

Comment (chiarazampolli) [reply Framework/Core/src/WorkflowCustomizationHelpers.cxx:61]: Ok with me, then.

**Comment by chiarazampolli** [comment]
Hi @shahor02 , @davidrohr ,

could you take a look and merge if you agree too?

chiara

**Pull request merged**